rpm/SPECS/openresty-openssl.spec

```diff
@@ -67,18 +67,21 @@ rm -rf %{buildroot}
 %files
 %defattr(-,root,root,-)
 
-%attr(0755,root,root) %{openssl_prefix}/lib/*.so*
-%attr(0755,root,root) %{openssl_prefix}/lib/*/*.so*
-
+%attr(0755,root,root) %{openresty_prefix}/bin/openssl
+%attr(0755,root,root) %{openresty_prefix}/%{_lib}/*.so*
+%attr(0755,root,root) %{openresty_prefix}/%{_lib}/*/*.so*
+%exclude %{openresty_prefix}/bin/c_rehash
+%exclude %{openresty_prefix}/misc
+%exclude %{openresty_prefix}/openssl.cnf
 
 %files devel
 %defattr(-,root,root,-)
 
-%{openssl_prefix}/include/*
-%{openssl_prefix}/lib/pkgconfig/*
-%attr(0755,root,root) %{openssl_prefix}/lib/*.a
+%{openresty_prefix}/include/*
+%{openresty_prefix}/%{_lib}/pkgconfig/*
+%attr(0755,root,root) %{openresty_prefix}/%{_lib}/*.a
```

Comment:
This openssl library should be installed into the `%{openssl_prefix}` location. Please do not change that.